firmware/shared/src/io/io_i2c_potentiometer.c

```diff
@@ -0,0 +1,23 @@
+#include "io_i2c_potentiometer.h"
+#include "hw_i2c.h"
+
+const uint8_t TARGET_ADDRESS = 0b0101111;
+
+static I2cInterface *pot;
```

Comment:
bro literally did not hear my sumrath roast session

Reply:
which line